install_process.go

```diff
@@ -63,18 +63,23 @@ func (ip YarnInstallProcess) ShouldRun(workingDir string, metadata map[string]in
 		return true, "", fmt.Errorf("failed to execute yarn config output:\n%s\nerror: %s", buffer.String(), err)
 	}
 
-	configFile, err := createTempFileAndWriteContents(buffer.Bytes(), "config-file")
+	nodeEnv := os.Getenv("NODE_ENV")
+	_, err = buffer.WriteString(nodeEnv)
 	if err != nil {
```

Comment:
Looks like `(*Buffer) WriteString` always returns nil. Could you please remove the error check?